README.md

```diff
@@ -45,7 +48,26 @@ pip install morphology-workflows
 
 ## Usage
 
-This workflow is based on the ``luigi`` library but can be run via the command line interface:
+### Create inputs
+
+The inputs should consist in:
+
+* a directory containing the input morphologies.
+* a CSV file with the following columns:
+    1. ``index``: the row index.
+    2. ``morph_path``: the path to the morphology file.
+    3. ``morph_name``: the name of the morphology.
+    4. ``brain_region``: the brain region in which the morphology is located.
+* a ``luigi.cfg`` file containing the configuration for all the tasks.
+* a ``logging.conf`` file containing the logging configuration.
```

Comment:
maybe add: "if one prefers default, just remove this file and uncomment line in luigi.cfg" ?

Reply:
Good point